harness/determined/common/experimental/experiment.py

```diff
@@ -166,6 +168,34 @@ def delete(self) -> None:
         """
         bindings.delete_DeleteExperiment(self._session, experimentId=self._id)
 
+    def download_code(self, output_dir: Optional[str] = None) -> str:
+        """Downloads a zipped tarball (*.tar.gz) of the experiment's submitted code locally.
+
+        If a file with the same name already exists in the output directory, overwrites the file.
```

Comment:
Could you please document the `exp-id_model_def.tar.gz` name? Then you can refer to that name in the "if a file..."

Reply:
good call, don't know why i didn't think to do that. done.